.github/workflows/build-&-publish-docker-image.yml

```diff
@@ -53,15 +71,33 @@ jobs:
           docker tag blobber:latest ${BLOBBER_REGISTRY}:$TAG-$SHORT_SHA
           docker push ${BLOBBER_REGISTRY}:$TAG-$SHORT_SHA  || { sleep 10 && docker push ${BLOBBER_REGISTRY}:$TAG-$SHORT_SHA; }
 
-  build_and_publish_validator:
```

Comment:
was this not more descriptive?

Reply:
because it is shown as `Build & Publish Docker Image / build_and_publish_validator (pull_request)`. I think `Build & Publish Docker Image / validator` is enough for readable.

Reply:
![Uploading Screen Shot 2021-12-30 at 23.20.59.png…]()